workspaces/topology/plugins/topology/src/translations/ref.ts

```diff
@@ -0,0 +1,340 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { createTranslationRef } from '@backstage/core-plugin-api/alpha';
+
+// CRITICAL: Export messages separately for testing
+export const topologyMessages = {
+  page: {
+    title: 'Topology',
+    subtitle: 'Kubernetes workload topology visualization',
+  },
+  toolbar: {
+    cluster: 'Cluster',
+    selectCluster: 'Select Cluster',
+    displayOptions: 'Display options',
+    currentDisplayOptions: 'Current display options',
+  },
+  emptyState: {
+    noResourcesFound: 'No resources found',
+    noResourcesDescription:
+      'No Kubernetes resources were found in the selected cluster.',
+  },
+  permissions: {
+    missingPermission: 'Missing Permission',
+    missingPermissionDescription:
+      'To view Topology, your administrator must grant you {{permissions}} {{permissionText}}.',
+    permission: 'permission',
+    permissions: 'permissions',
+    goBack: 'Go back',
+  },
+  sideBar: {
+    details: 'Details',
+    resources: 'Resources',
+    logs: 'Logs',
+    events: 'Events',
+    overview: 'Overview',
+    pods: 'Pods',
+    services: 'Services',
+    routes: 'Routes',
+    ingresses: 'Ingresses',
+    configMaps: 'ConfigMaps',
+    secrets: 'Secrets',
+    persistentVolumeClaims: 'Persistent Volume Claims',
+  },
+  workload: {
+    deployment: 'Deployment',
+    daemonSet: 'DaemonSet',
+    statefulSet: 'StatefulSet',
+    job: 'Job',
+    cronJob: 'CronJob',
+    pod: 'Pod',
+    virtualMachine: 'Virtual Machine',
+    replicaSet: 'ReplicaSet',
+  },
```

Comment:
Not used yet?

Reply:
Yeah, you are right, these we are using directly from kubernetes api returned `kind`. No need to translate them and they are not used anywhere in the code. Good catch 👍